### PR TITLE
[Resources] Fix deployment validate response (urgent)

### PR DESCRIPTION
**Pull request by filizt**
This PR fixes a bug in the \validate path responses, marking it as async and long running operation. This is the current service behavior.

</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json [hunk 1]
```diff
@@ -586,6 +594,12 @@
               "$ref": "#/definitions/DeploymentValidateResult"
             }
           },
+          "202": {
+            "description": "Accepted - The request has been accepted for processing and the operation will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/DeploymentValidateResult"
```
Comment (Tiano2017) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json:600]: A 202 response would not contain payload with it.

Comment (filizt) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json:600]: Updating now.

File: specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json [hunk 2]
```diff
@@ -285,6 +286,12 @@
...
```
Comment (marcre) [inline specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json]: Validate response or empty?  The other 202s seem to have no schema listed.

Comment (filizt) [reply specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json]: No schema needed. Updated in the last iteration. 

---
In reply to: [406609375](https://github.com/Azure/azure-rest-api-specs/pull/9007#discussion_r406609375) [](ancestors = 406609375)

**Comment by lirenhe** [comment]
@filizt, as your PR would change the behavior of the SDK, do you have the sdk already generated already?

**Review by marcre — APPROVED** [review]

**Comment by filizt** [comment]
> @filizt, as your PR would change the behavior of the SDK, do you have the sdk already generated already?

Yes, we have the SDK generated and it's in PR. See here: https://github.com/Azure/azure-sdk-for-net/pull/11240

**Review by lirenhe — APPROVED** [review]

**Pull request merged**